doc/start/intro.rst

```diff
@@ -9,10 +9,10 @@ System`.  All :term:`Ceph Storage Cluster` deployments begin with setting up
 each :term:`Ceph Node` and then setting up the network. 
 
 A Ceph Storage Cluster requires the following: at least one Ceph Monitor and at
-least one Ceph Manager, and at least as many Ceph OSDs as there are copies of
-an object stored on the Ceph cluster (for example, if three copies of a given
-object are stored on the Ceph cluster, then at least three OSDs must exist in
-that Ceph cluster). 
+least one Ceph Manager, and at least as many :term:`Ceph Object Storage
+Daemon<Ceph OSD>`\s (OSDs) as there are copies of a given object stored on the
+Ceph cluster (for example, if three copies of a given object are stored on the
+Ceph cluster, then at least three OSDs must exist in that Ceph cluster).
```

Comment:
I would favor "store in the"

Reply:
Do you mean "[...]if you store three copies of a given object in the Ceph cluster[...]"?

Reply:
of a given object are stored IN the Ceph cluster

Reply:
Accepted.